src/plugins/query_enhancements/public/query_assist/components/query_assist_summary.tsx

```diff
@@ -246,38 +256,42 @@ export const QueryAssistSummary: React.FC<QueryAssistSummaryProps> = (props) =>
                     })}
                   />
                 </EuiFlexItem>
-                <EuiFlexItem grow={false}>
-                  <EuiSmallButtonIcon
-                    aria-label="feedback thumbs up"
-                    color="text"
-                    iconType="thumbsUp"
-                    title={
-                      !feedback
-                        ? i18n.translate('queryEnhancements.queryAssist.summary.goodResponse', {
-                            defaultMessage: `Good response`,
-                          })
-                        : afterFeedbackTip
-                    }
-                    onClick={() => onFeedback(true)}
-                    data-test-subj="queryAssist_summary_buttons_thumbup"
-                  />
-                </EuiFlexItem>
-                <EuiFlexItem grow={false}>
-                  <EuiSmallButtonIcon
-                    aria-label="feedback thumbs down"
-                    color="text"
-                    title={
-                      !feedback
-                        ? i18n.translate('queryEnhancements.queryAssist.summary.badResponse', {
-                            defaultMessage: `Bad response`,
-                          })
-                        : afterFeedbackTip
-                    }
-                    iconType="thumbsDown"
-                    onClick={() => onFeedback(false)}
-                    data-test-subj="queryAssist_summary_buttons_thumbdown"
-                  />
-                </EuiFlexItem>
+                {feedback !== FeedbackStatus.THUMB_DOWN && (
+                  <EuiFlexItem grow={false}>
+                    <EuiSmallButtonIcon
+                      aria-label="feedback thumbs up"
+                      color={feedback === FeedbackStatus.THUMB_UP ? 'subdued' : 'text'}
+                      iconType="thumbsUp"
+                      title={
+                        !feedback
+                          ? i18n.translate('queryEnhancements.queryAssist.summary.goodResponse', {
+                              defaultMessage: `Good response`,
+                            })
+                          : afterFeedbackTip
+                      }
+                      onClick={() => onFeedback(true)}
+                      data-test-subj="queryAssist_summary_buttons_thumbup"
```

Comment:
can we disable the button when `feedback === FeedbackStatus.THUMB_UP`, so that we don't need to check `if (feedback !== FeedbackStatus.NONE) return;` for `onFeedback()`